src/settings.py

```diff
@@ -52,6 +52,43 @@
 PROVIDER_ROLE = 'seller'
 CUSTOMER_ROLE = 'customer'
 
+CHARGE_PERIODS = [
```

Comment:
Why not saving this as a dict? This way you don't need to iterate to get the values